### ISSUE TITLE
right click an item goes in personal bank

### ISSUE THREAD

**Issue by Detro84** [arkayenro/arkinventory#1951]
**What version number you are using?**
alpha

**What game client are you playing? Retail, Burning Crusade, Classic / Live, PTR, Beta**
retail

**What language is the game client set to?**
English

**Ask your question**
If I right click an item, it comes in my personal bank, I need to turn off arkinventory and use the blizzard ui to get them to the warbank

**Comment by arkayenro** [comment]
yes, that doesnt work yet because you cant select a specific warbank tab so it cant tell where you want it to go

**Comment by arkayenro** [comment]
actually this appears to be a bug.  the default ui does it as well.  im not sure i can code around it but i'll try

**Comment by Detro84** [comment]
thx! And arkinventory doesn't work with tsm, i need to turn off arkinventory to get tsm pick my mats from the warbandbank

**Comment by arkayenro** [comment]
i dont use tsm.  i'd need an error of some sort to try and work it out.  its possibly because i didnt used to change things in the default bank ui to align them as it wasnt needed before.

**Comment by hollo6** [comment]
I just came here to report the same thing, but disabling AI makes right-click transfer the item into the warbank as expected so it's not a blizzard bug.

**Comment by hollo6** [comment]
Nevermind I just installed alpha-29 and this one behaves alright in that regard but now how do I view my personal bank like before? I need to be able to see my personal bank and reagent bank together.

**Comment by Detro84** [comment]
oh wauw @arkayenro thx man!

**Comment by Detro84** [comment]
hmm did you remove the display on/off function? Now I only can see 1 tab at a time?

**Comment by arkayenro** [comment]
where right clicked items in your bags end up going to in the bank is now highlighted so you can tell more easily.

normally the panels would keep them apart but where you have, for example, combined the bank and regent bank and you want things to go into the reagent bank, you need to click on that slot first so it gets highlighted.  clicking the bank or a bank bag will highlight the bank.

the account banks are slightly different in that the item will end up in the first tab, or it could be based on the cleanup filters (not really sure), but it will end up in the account bank if you have one of those highlighted.

**Comment by Detro84** [comment]
now he sorts everuthing in the bars you set, but he doesn't sort the items in the bars, everything is scrambled and refresh doesn't do anything

**Comment by arkayenro** [comment]
im not seeing any obvious sort issues with my config

in edit mode click on a bar number to check what sort method is assigned, and if it has an override set

**Comment by Detro84** [comment]
strange, the sorting method is the defaul bag/slots, but like here the true iron ore is before the blackrock ore, I can manually override with the 'sort by name' method, I don't know how the default bag/slots sort?
![image](https://github.com/user-attachments/assets/c22fed83-94dd-4fb9-8cef-3d992607274c)

**Comment by Detro84** [comment]
my bad, I had allways the default and I didn't understand why some items where not sorted, now I set every bag on 'Name (ascending)'

**Comment by Detro84** [comment]
hmm my rule (grim-veiled) did not work anymore, any idea why? 
![image](https://github.com/user-attachments/assets/abd9aa64-a13b-405c-a2c7-a6cb3c0717f4)

**Comment by Detro84** [comment]
none of my sl lego rules work anymore, I see shadowghast items in my umbrahide group

**Comment by arkayenro** [comment]
the window settings youre using has bag/slot as the default sort method so its not going to sort by name.

if you want everything to be sorted by name as the default then you would need to update the settings

are you sure youre using the correct profile/blueprints?

**Comment by Detro84** [comment]
no issue with the rules, just 'user error' :)
